quickstarts/redmine/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: redmine
+description: |
+  ## Comprehensive monitoring quickstart for Redmine
+  We can monitor your Redmine application in a complex environment and keep you on top of performance with our Ruby agent.
+  ## Why monitor Redmine?
+  The performance monitoring tool for Redmine helps you monitor the entire infrastructure and identifies the critical issues in your Redmine application. Redmine performance monitoring enables you to get a complete overview of Redmine errors and know when and where your Redmine application slows down. Some key components are:
+  - Helps you identify issues that are truly important and quickly resolve them. Gain knowledge of performance issues in advance and thus giving you ample time to avoid application downtime.
+  - Performance provides you with the information you need to not only prioritize critical performance issues, but also trace them down to the root cause and solve them more quickly.
+  - Monitor your issues related to projects which are created in the Redmine application.
+
+  ## What’s included in this quickstart?
+  New Relic's Redmine monitoring quickstart boasts instant full-stack observability out-of-the-box:
```

Comment:
"You'll get instant full-stack observability of your Redmine app out-of-the-box:"